src/App.tsx

```diff
@@ -15,14 +17,21 @@ import ImageMapper, { ImageMapperProps, MapAreas } from 'react-img-mapper';
 import { AllWorlds, IdToMapObject } from "./config/map-details";
 import { MapDetails } from "./config/types";
 import { Stack } from "./utils/MapStack";
+import Alert from '@mui/material/Alert';
+import Button from '@mui/material/Button'; 
+import ButtonGroup from '@mui/material/ButtonGroup';
 
 
 function App() {
   const [currentQuestion, setCurrentQuestion] = useRecoilState(currentQuestionState);
   const [videoShown, setVideoShown] = useRecoilState(videoShownState);
   const [world, setWorld] = useState<Stack<MapDetails>>(new Stack<MapDetails>(5, AllWorlds));
   const [hoverArea, setHoverArea] = useState<String>();
-  const [result, setResult] = useState<Boolean>();
+  const [result, setResult] = useState<String>();
+  const [answer, setAnswer] = useState<String>("");
+  const [gameStarted, ] = useRecoilState(gameStartedState);
```

Comment:
Can update this to use https://recoiljs.org/docs/api-reference/core/useRecoilValue:
```
const gameStarted = useRecoilValue(gameStartedState);
```